solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/pom.xml

```diff
@@ -106,6 +106,15 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-actuator</artifactId>
+        </dependency>
```

Comment:
Make this optional in the core as we've done in the other binder artifact: https://github.com/SolaceProducts/solace-spring-cloud/blob/master/solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder/pom.xml#L53-L57

Reply:
Going forward we wont have a single artifact be _optional_, but modularize better and use dependency management for avoidance of unnecessary dependencies. Compile time dependencies should not be marked "optional".

Reply:
After a careful consideration of all possible applicable options, we decided to follow with conservative approach and make `actuator` dependency optional.